### PR TITLE
check my objects programs

### PR DESCRIPTION
**Pull request by Rahul7808**
@Nilesh28v, check my code and comment

### CODE REVIEW

File: 22-2-22objects/obj.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const person = {
+    fName : 'Ram',
+    fName:'sss',
+    lName : 'Manohar',
+    age : 31,
+    gender : 'Male',
+    city : 'Hyd',
+    isPassed : true,
+    fullName : function(street) {
+        return `full name is ${this.fName} ${this.lName} from ${street}`
```
Comment (Nilesh28v) [inline 22-2-22objects/obj.js:12]: you can mention why you're using 'from' here

File: 22-2-22objects/rest2.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// multiply values by rest 
+function multiply(multiplier, ...args) {
+    return args.map(element => {
+      return multiplier * element
+    })
+  }
+  
+  let arr = multiply(2, 15, 25, 42)
```
Comment (Nilesh28v) [inline 22-2-22objects/rest2.js:8]: you can mention value 2 is going into multiplier and other values are going into ...args.

**Review by Nilesh28v — APPROVED** [review]

**Review by Rahul7808 — COMMENTED** [review]
I will change.

**Comment by Rahul7808** [comment]
Thanks

**Pull request merged**